nixos/modules/services/web-apps/wordpress.nix

```
@@ -168,7 +191,7 @@ let
 
           name = mkOption {
             type = types.str;
-            default = "wordpress";
+            default = if lib.versionAtLeast config.system.stateVersion "21.11" then "wordpress-${name}" else "wordpress";
```

Maybe I should change this to
```suggestion
            default = if lib.versionAtLeast config.system.stateVersion "21.11" then "wordpress-${replaceStrings [ "." ] [ "_" ] name}" else "wordpress";
```